general.css

```diff
@@ -0,0 +1,918 @@
+@font-face {
+  font-family: 'et-line';
+  src: url('et-line-font/fonts/et-line.eot');
+  src: url('et-line-font/fonts/et-line.eot?#iefix') format('embedded-opentype'), url('et-line-font/fonts/et-line.woff') format('woff'), url('et-line-font/fonts/et-line.ttf') format('truetype'), url('et-line-font/fonts/et-line.svg#et-line') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+[data-icon]:before {
+  font-family: 'et-line';
+  content: attr(data-icon);
+  speak: none;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  display: inline-block;
+  font-size: 2rem;
+}
+
+:root {
+  --first-font-color:#ffffff;
+  --section-gray-color:#f8f8f8;
+  --hover-color:#5dc5ab;
+  --section-title-color: #494848;
+  --section-paragraph-color: #8b8787;
+  --title-size: 1.9rem;
+  --paragraph-size: 0.8rem;
+  --subtitle-size: 1.3rem;
+  --mobile-margin: 0 1rem;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: auto;
+  font-family: 'Oxygen', sans-serif;
+}
+
+.icon {
+  display: inline-block;
+  width: 1em;
+  height: 1rem;
+  fill: var(--first-font-color);
+  cursor: pointer;
+}
+
+.main-title {
+  color: var(--first-font-color);
+  text-align: center;
+  margin: 0;
+  padding: 1rem 0.5rem;
+}
+
+.main-paragraph {
+  color: var(--first-font-color);
+  text-align: center;
+  margin: 0;
+  padding: 1rem 0.5rem;
+}
+
+.carrousel-images {
+  background: url('images/home-bg.jpg') no-repeat;
+  height: 70vh;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.firstSlide,
+.secondSlide {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+}
+
+.firstSlide {
+  background: url('images/home-bg.jpg') left 20% top 30% / cover no-repeat;
+  animation: fadeFirstSlide 10s infinite;
+  -webkit-animation: fadeFirstSlide 10s infinite;
+}
+
+.secondSlide {
+  background: url('images/home-bg-2.jpg') left 20% top 30% / cover no-repeat;
+  animation: fadeSecondSlide 10s infinite;
+  -webkit-animation: fadeSecondSlide 10s infinite;
+}
+
+.text-header {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: transparent;
+  z-index: 1;
+}
+
+@keyframes fadeFirstSlide {
```

Comment:
this name doesn't describe what the animation does, it describes where it is applied